.github/workflows/tox.yml

```diff
@@ -12,8 +12,17 @@ jobs:
               python-version: 3.6
             - environment: py38
               python-version: 3.8
+            - environment: py310
+              python-version: 3.10
             - environment: pep8
               python-version: 3.8
+            - environment: pep8
+              python-version: 3.10
+          exclude:
+            - python-version: 3.10
+              if: ${{ github.ref != 'refs/heads/stackhpc/zed' }}
```

Comment:
I think this would only work if running a workflow against the branch. It won't work for PRs.

> The fully-formed ref of the branch or tag that triggered the workflow run. For workflows triggered by push, this is the branch or tag ref that was pushed. For workflows triggered by pull_request, this is the pull request merge branch. For workflows triggered by release, this is the release tag created. For other triggers, this is the branch or tag ref that triggered the workflow run. This is only set if a branch or tag is available for the event type. The ref given is fully-formed, meaning that for branches the format is refs/heads/<branch_name>, for pull requests it is refs/pull/<pr_number>/merge, and for tags it is refs/tags/<tag_name>. For example, refs/heads/feature-branch-1.

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
github.base_ref should work for PRs. I'm not sure whether we use this workflow for anything else, but perhaps `github.base_ref || github.ref` would work?